services/symfony/symfony-insight.spec.js

```diff
@@ -0,0 +1,255 @@
+'use strict'
+
```

Comment:
This makes things a lot cleaner 👍 Haven't checked out this branch locally yet, but I assume the coverage levels are still the same for these services?

Reply:
I’m hoping the coverage is good via test:core now! I don’t usually run coverage locally, I’m curious though.

Reply:
It may be even higher now. Coverage was pretty high before but I think there were a couple blocks in the base service that I didn't bother to hit, and those blocks may be gone now (covered via test:core like you noted).